libs/cli/langgraph_cli/docker.py

```diff
@@ -117,11 +105,36 @@ def check_capabilities(runner) -> DockerCapabilities:
     )
 
 
+def debugger_compose(*, port: Optional[int] = None, host: Optional[str] = None) -> str:
+    if port is None:
```

Comment:
IMO this is not very intuitive. could we do this check on host instead?

- cli option for host: no default in `click`
- cli option for port: default to `DEFAULT_PORT` in `click`

and below add environment only `if host != localhost`?

Reply:
This is not really for changing the host of the debugger image, rather to change the default URL used by the debugger to access the LangGraph API. 

Renamed the CLI flag to closely match the intent.

Reply:
ah, that makes sense, thanks for clarifying!